src/commands/task-groups/create.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2023 Relay Inc.
+
+import * as fs from 'fs'
+
+import { Command } from '../../lib/command'
+import * as flags from '../../lib/flags'
+// eslint-disable-next-line quotes
+import debugFn = require('debug')
+import { taskGroupCreateArgs, CreateTaskGroupArgs, createTaskGroupArgs } from '../../lib/args'
+import { createTaskGroup } from '../../lib/tasks'
+
+const debug = debugFn(`task-groups:create`)
+
+export default class TaskGroupsCreateCommand extends Command {
+  static description = `Create a task group`
+  static strict = false
+
+  // static hidden = true
+
+  static flags = {
+    ...flags.subscriber,
+  }
+
+  static args = [
+    ...taskGroupCreateArgs
+  ]
+
+  async run(): Promise<void> {
+    const { flags, argv } = await this.parse(TaskGroupsCreateCommand)
+    const subscriberId = flags[`subscriber-id`]
+    const taskGroupArgs: CreateTaskGroupArgs = createTaskGroupArgs(argv)
+
+    let encoded_string = taskGroupArgs.members as string
+    if (encoded_string.charAt(0) == `@`) {
```

Comment:
Should we guard on file size? As in... test to see what the OS reports as the files size and limit based on that?